lib/foods.js

```diff
@@ -21,33 +21,41 @@ function checkInputs() {
     var calories = $('#calories-field input').val();
     var foodsRaw = sessionStorage.getItem("foods");
     var foods = $.parseJSON(foodsRaw);
-    foodTableSingleRow(food, calories, true, false, true)
-    saveFoodList();
+    foodTableSingleRow($('#food-list'), food, calories, true, false, true);
+    saveFoodList($('#food-list'));
     $('#name-field input').val("");
     $('#calories-field input').val("");
   };
 };
 
-function foodTableSingleRow(name, calories, deleteButton, checkBox, addToList){
+function foodTableSingleRow(whichTable, name, calories, deleteButton, checkBox, addToList){
```

Comment:
I'm kind of picking nits, but I think a better name might be `renderFoodTableSingleRow` or `foodTableSingleRowHTML`, just to add the context that you're returning HTML, and not like an instance of a row in JS.